resources/qml/Preferences/GeneralPage.qml

```diff
@@ -72,6 +72,9 @@ UM.PreferencesPage
         var defaultTheme = UM.Preferences.getValue("general/theme")
         setDefaultTheme(defaultTheme)
 
+        UM.Preferences.resetPreference("cura/single_instance")
+        scaleToFitCheckbox.checked = boolCheck(UM.Preferences.getValue("cura/single_instance"))
```

Comment:
I will fix this one for you :) (I'm fairly sure this should be singleInstanceCheckbox.checked)

Reply:
It should, thanks!